### PR TITLE
proto,scripts(ticdc): generate protobuf with gogofaster

### LINKED ISSUE #5256: Generate protobuf code with gogofaster
As title

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve?

Issue Number: close #5256

### What is changed and how it works?

Generate protobuf code with gogofaster.
Also fixed a bug that fails to generate CDCPeerToPeer.proto.

```shell
➜ ./generate-proto.sh 
generate canal & craft benchmark protocol code...
gogoproto/gogo.proto: File not found.
CDCPeerToPeer.proto:18:1: Import "gogoproto/gogo.proto" was not found or had errors.
```

### Check List 

Tests 

 - Manual test

Code changes

 - Has exported function/method change
 - Has exported variable/fields change

### Release note 

```release-note
None
```

### CODE REVIEW

File: scripts/download-protoc.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# Copyright 2021 PingCAP, Inc.
```
Comment (overvenus) [inline scripts/download-protoc.sh:2]:
```suggestion
# Copyright 2022 PingCAP, Inc.
```

File: scripts/generate-protobuf.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Copyright 2021 PingCAP, Inc.
```
Comment (overvenus) [inline scripts/generate-protobuf.sh:2]:
```suggestion
# Copyright 2022 PingCAP, Inc.
```

File: scripts/download-protoc.sh [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+		-o $TOOLS_INCLUDE_DIR/gogoproto/gogo.proto
+
+echo "download protoc..."
+[ ! -d $TOOLS_BIN_DIR ] && mkdir -p $TOOLS_BIN_DIR
+[ ! -f $TOOLS_BIN_DIR/protoc ] &&
+	mkdir -p /tmp/cdc/protoc &&
+	curl -sL $PROTOC_URL -o /tmp/cdc/protoc/protoc-$PROTOC_VERSION-linux-x86_64.zip &&
+	unzip -q -o -d /tmp/cdc/protoc /tmp/cdc/protoc/protoc-$PROTOC_VERSION-linux-x86_64.zip &&
+	mv /tmp/cdc/protoc/include/google $TOOLS_INCLUDE_DIR &&
+	mv /tmp/cdc/protoc/bin/protoc $TOOLS_BIN_DIR/protoc
```
Comment (hi-rustin) [inline scripts/download-protoc.sh:53]: Echo a success message?

**Review by hi-rustin — APPROVED** [review]
Can you tell us about your motivation?

**Review by 3AceShowHand — APPROVED** [review]

**Pull request merged**